sdk/keyvault/keyvault-admin/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "@azure/keyvault-admin",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "4.1.0-preview.2",
```

Comment:
Version should be 1.0.0-preview.1 ?

Reply:
.Net is aligning with the versions in the other of their clients: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md#410-preview1-unreleased